opendistro-for-elasticsearch/README.md

```diff
@@ -0,0 +1,18 @@
+# elasticsearch-odfe
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Service package
+
+## Usage
+
+```bash
+hab pkg install core/opendistro-for-elasticsearch
+hab svc load core/opendistro-for-elasticsearch
+
+curl -sk https://admin:admin@${addr}
+```
```

Comment:
Be sure to follow the [readme template](https://github.com/habitat-sh/core-plans/blob/master/README_TEMPLATE_FOR_PLANS.md) as much as possible. Specifically for service plans, we expect some information around what topologies, update strategies and monitoring capabilities are available for the service. A bare minimum example can be found on [elasticsearch5](https://github.com/habitat-sh/core-plans/blob/master/elasticsearch5/README.md).